service/router.go

```diff
@@ -46,6 +46,9 @@ func AdminRouter(env *Env) *mux.Router {
 	// Start the web service router
 	router := mux.NewRouter()
 
+	// API routes: csrf token
+	router.Handle("/v1/csrf", Middleware(http.HandlerFunc(TokenHandler), env)).Methods("GET")
```

Comment:
sounds to me quite generic having "token" in the path.. token of what? a nonce is also a token. makes sense /v1/csrftoken ?